src/pykeen/evaluation/evaluator.py

```diff
@@ -464,6 +467,7 @@ def evaluate(
     model: Model,
     mapped_triples: MappedTriples,
     evaluators: Union[Evaluator, Collection[Evaluator]],
+    additional_pos_triples: Optional[MappedTriples] = None,
```

Comment:
why not just cat the mapped_triples and additional_pos_triples together before calling this function?

Reply:
We need to  distingusih between `mapped_triples `and `additional_pos_triples`, because `mapped_triples` are the triple to be evaluated.

Reply:
Thanks for clarifying. I think it's time to write proper docs in the `Evaluator.__init__()` to go along with this PR so that's more clear if you're looking at just this spot (It's written again below, but what are you gonna do when your PR reviewer is a lazy guy). I'm not a huge fan of this argument name either so maybe we can think of something better

Reply:
Feel free to change the argument name :)

Reply:
@mali-git @cthoyt What about `additional_filter_triples`?

Reply:
better ^
thank you